systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java

```diff
@@ -78,21 +53,11 @@ public static long getTimeoutForResourceDeletion() {
     }
 
     public static long getTimeoutForResourceDeletion(String kind) {
-        long timeout;
-
-        switch (kind) {
-            case Kafka.RESOURCE_KIND:
-            case KafkaConnect.RESOURCE_KIND:
-            case KafkaMirrorMaker2.RESOURCE_KIND:
-            case KafkaBridge.RESOURCE_KIND:
-            case TestConstants.STATEFUL_SET:
-            case TestConstants.POD:
-                timeout = Duration.ofMinutes(5).toMillis();
-                break;
-            default:
-                timeout = Duration.ofMinutes(2).toMillis();
-        }
-
-        return timeout;
+        return switch (kind) {
+            case Kafka.RESOURCE_KIND, KafkaConnect.RESOURCE_KIND, KafkaMirrorMaker2.RESOURCE_KIND,
+                 KafkaBridge.RESOURCE_KIND, TestConstants.STATEFUL_SET, TestConstants.POD ->
+                    Duration.ofMinutes(5).toMillis();
+            default -> Duration.ofMinutes(2).toMillis();
```

Comment:
For me it's just stylistic change but each ResourceType in separate line would be better rather than multiple resources in one line (at least for me it improves readability). => that applies on multiple enums you have changed. Not sure what others think about that.

Reply:
Yeah, I guess in terms of readability, it would be better to have it on separate lines.

Reply:
Thanks for the feedback. I updated the formatting of the switch blocks